Assets/MixedRealityToolkit/_Core/Interfaces/BoundarySystem/IMixedRealityBoundarySystem.cs

```diff
@@ -2,39 +2,61 @@
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
 using Microsoft.MixedReality.Toolkit.Internal.Definitions.Utilities;
+using Microsoft.MixedReality.Toolkit.Internal.Interfaces.Events;
 using UnityEngine;
 using UnityEngine.Experimental.XR;
 
-namespace Microsoft.MixedReality.Toolkit.Internal.Interfaces
+namespace Microsoft.MixedReality.Toolkit.Internal.Interfaces.BoundarySystem
 {
     /// <summary>
     /// Manager interface for a Boundary system in the Mixed Reality Toolkit
     /// All replacement systems for providing Boundary functionality should derive from this interface
     /// </summary>
-    public interface IMixedRealityBoundarySystem : IMixedRealityManager
+    public interface IMixedRealityBoundarySystem : IMixedRealityEventSystem, IMixedRealityEventSource
     {
         /// <summary>
         /// The scale (ex: World Scale) of the experience.
         /// </summary>
         ExperienceScale Scale { get; set; }
 
         /// <summary>
-        /// The height of the playspace, in meters.
+        /// The height of the play space, in meters.
         /// </summary>
         /// <remarks>
         /// This is used to create a three dimensional boundary volume.
         /// </remarks>
         float BoundaryHeight { get; set; }
 
         /// <summary>
-        /// Enable / disable the platform's playspace boundary rendering.
+        /// Enable / disable boundary wall rendering.
+        /// </summary>
```

Comment:
let's comment out the xml here too please :)

Reply:
ok

Reply:
made this change... not sure why GitHub didn't notice.